server.go

```diff
@@ -76,6 +77,9 @@ func (server *Server) Serve(param ServerParam) {
 		fmt.Printf("master slides can be found at http://*:%d/master/\n", port)
 	}
 
+	if !param.DisableAutoOpen {
+		go openBrowser(port)
```

Comment:
@king-11 

Lint throwing an error:
https://github.com/yusukebe/revealgo/runs/5000226417?check_suite_focus=true#step:4:69

Please check it.

Reply:
Made the changes